vega/widget.py

```diff
@@ -1,13 +1,19 @@
 from __future__ import print_function
 
-import contextlib
 import json
-import uuid
-import sys
+import pandas as pd
+from ._frontend import module_name, EXTENSION_SPEC_VERSION
 
+# from ipytablewidgets.compressors import *
+from ipytablewidgets import (serialization,
+                             SourceAdapter,
+                             PandasAdapter,
+                             NumpyAdapter, TableType)
+
+import numpy as np
 try:
-    from ipywidgets import DOMWidget
-    from traitlets import Unicode, Dict
+    import ipywidgets as widgets
```

Comment:
I think this is clearer

```suggestion
    import ipywidgets
```